src/operator/LimitsInput.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {FC} from 'react'
+import {Input, InputType} from '@influxdata/clockface'
+import {OrgLimits} from 'src/types/operator'
+
+interface Props {
+  type: InputType
+  name: string
+  limits: OrgLimits
+  onChangeLimits: (limits: OrgLimits) => void
+}
+
+const LimitsInput: FC<Props> = ({type, name, limits, onChangeLimits}) => {
+  const onChange = e => {
+    const newValue =
+      type === InputType.Number ? parseFloat(e.target.value) : e.target.value
+
+    const newLimits = {...limits, [name]: newValue}
+    onChangeLimits(newLimits)
+  }
+  return (
+    <Input
+      type={type}
+      name={name}
+      value={limits?.[name] ?? ''}
```

Comment:
This code uses lodash's `get` and `set` here because `name` is generally a dotted name like `rate.readKBs` or similar. I don't believe that this code (and line 17 above) will do the same thing in that case.